etc/headless-client.api.md

```diff
@@ -5,7 +5,7 @@
 ```ts
 
 import { AxiosResponse } from 'axios';
-import FormData from 'form-data';
+import { default as FormData_2 } from 'form-data';
```

Comment:
Question: Why is this necessary? Why not just `import { default as FormData } from 'form-data';`
Mostly just curious why its necessary to do :)

Reply:
It's the auto generated documentation not sure why it's changed, but I did update some depenencies recently so that might be why